### ISSUE TITLE
Zoom doesn't work on Microsoft surface RT and PRO 

### ISSUE THREAD

**Issue by michaelthe** [cubiq/iscroll#870]
While trying to zoom on a surface RT devise the browser zooms the whole page instead of the iscroll.
this also happens on surface PRO with firefox.

**Comment by starquake** [comment]
Yeah would be great if iScroll could support Windows on the Surface devices

**Comment by alvaro0555** [comment]
The solution in this answer might help: https://github.com/cubiq/iscroll/issues/583

**Comment by starquake** [comment]
No that doesn't help. Zoom still doesn't work. I understand you don't have a device to test it on? I will have a look if I can find some time to fix it and send you a pull request.
